go/interpreter/lfvm/instructions_test.go

```diff
@@ -1291,3 +1291,61 @@ func TestInstructions_EIP2929_SSTOREReportsOutOfGas(t *testing.T) {
 		}
 	}
 }
+
+func TestInstructions_JumpInstructionsCheckJUMPDEST(t *testing.T) {
```

Comment:
For a follow-up: a related test that should be added for conditional jumps: if the condition is false, it must not test the jump target for pointing to a JUMPDEST; invalid target destinations must be accepted if the branch is not taken.

Reply:
I think this is not that far off. I extend the scope of this PR.